modin/pandas/__init__.py

```diff
@@ -12,81 +12,82 @@
 # governing permissions and limitations under the License.
 
 import pandas
+import warnings
 
-__pandas_version__ = "1.3.5"
+__pandas_version__ = "1.4.0"
 
 if pandas.__version__ != __pandas_version__:
-    import warnings
-
     warnings.warn(
         "The pandas version installed {} does not match the supported pandas version in"
         " Modin {}. This may cause undesired side effects!".format(
             pandas.__version__, __pandas_version__
         )
     )
 
-from pandas import (
-    eval,
-    cut,
-    factorize,
-    test,
-    qcut,
-    date_range,
-    period_range,
-    Index,
-    MultiIndex,
-    CategoricalIndex,
-    bdate_range,
-    DatetimeIndex,
-    Timedelta,
-    Timestamp,
-    to_timedelta,
-    set_eng_float_format,
-    options,
-    Flags,
-    set_option,
-    NaT,
-    PeriodIndex,
-    Categorical,
-    Interval,
-    UInt8Dtype,
-    UInt16Dtype,
-    UInt32Dtype,
-    UInt64Dtype,
-    SparseDtype,
-    Int8Dtype,
-    Int16Dtype,
-    Int32Dtype,
-    Int64Dtype,
-    Float32Dtype,
-    Float64Dtype,
-    StringDtype,
-    BooleanDtype,
-    CategoricalDtype,
-    DatetimeTZDtype,
-    IntervalDtype,
-    PeriodDtype,
-    RangeIndex,
-    Int64Index,
-    UInt64Index,
-    Float64Index,
-    TimedeltaIndex,
-    IntervalIndex,
-    IndexSlice,
-    Grouper,
-    array,
-    Period,
-    show_versions,
-    DateOffset,
-    timedelta_range,
-    infer_freq,
-    interval_range,
-    ExcelWriter,
-    datetime,
-    NamedAgg,
-    NA,
-    api,
-)
+with warnings.catch_warnings():
```

Comment:
Why do we need this?

Reply:
pandas throws warnings for several imports that they will be deprecated soon. Every time there is an import these errors are thrown without suppressing them.